.github/workflows/ci-pr.yml

```diff
@@ -27,10 +27,6 @@ jobs:
       with:
         maven-version: 3.8.8
         maven-mirror: 'https://dlcdn.apache.org/maven/maven-3/'
-    - name: Setup Testcontainers Cloud Client
```

Comment:
Just curious why we are removing this configuration in this PR.

Reply:
CI PR doesn't have access to secrets.

Reply:
Understood.  We probably want to enumerate this in our commit message since it didn't fit the rest of the PR.

Reply:
This should just be in a separate commit (it can be in this PR).

Reply:
Changed update to ci-pr.yml to separate commit